### PR TITLE
import tests

### PR DESCRIPTION
**Pull request by ruchim**
Test sub_workflow cases with Server mode imports

### CODE REVIEW

File: src/main/scala/centaur/test/workflow/WorkflowData.scala [hunk 1]
```diff
@@ -20,8 +23,33 @@ object WorkflowData {
...
+          imports foreach { p =>
+            val srcFile = File(p.toAbsolutePath.toString)
+            val destFile = importsDir / srcFile.name
+            srcFile.copyTo(destFile, true) }
+
+          importsDir.zip()
+        }
+          //TODO: There's no mechanism to match to an empty list for imports yet, but that
+          //would enable testing of empty zip files for imports.
+        case Nil => File.newTempDir(File(wdl.toString).nameWithoutExtension + "_import").zip()
```
Comment (Horneth) [inline src/main/scala/centaur/test/workflow/WorkflowData.scala]: Could it return None instead ?

**Review by Horneth — APPROVED** [review]

**Comment by Horneth** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/117/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/recursive_imports.test [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    sub_workflow_hello_world/sub_workflow_hello_world_import.wdl,
+    sub_workflow_interactions/sub_workflow_interactions_import.wdl,
+    sub_workflow_interactions/sub_workflow_interactions.wdl
+  ]
+}
+
+metadata {
+  workflowName: recursive_imports
+  status: Succeeded
+}
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/recursive_imports.test]: newline

File: src/main/resources/standardTestCases/recursive_imports/recursive_imports.wdl [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+workflow recursive_imports {
+  call hello.sub.wf_hello {}
+  call forkjoin.join { input: wcCount=15, grepCount=10 }
+  call interactions.counter.countTo { input: value = 3 }
+
+  output {
+    wf_hello.*
+    forkjoin.join.*
+  }
+}
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/recursive_imports/recursive_imports.wdl]: newline

**Review by cjllanwarne — APPROVED** [review]
LGTM

**Comment by cjllanwarne** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/117/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**